models/index.js

```diff
@@ -0,0 +1,33 @@
+const Sequelize = require('sequelize')
+const allConfigs = require('../configs/sequelize')
+const manufacturersModel = require('./manufacturers')
+const vehicleModelsModel = require('./vehicleModels')
+const myCarsModel = require('./myCars')
+const myCars2VehiclesModel = require('./myCars2Vehicles')
+
+const environment = process.env.NODE_ENV || 'development'
+const config = allConfigs[environment]
+
+const connection = new Sequelize(config.database, config.username, config.password, {
+  host: config.host, dialect: config.dialect,
+})
+
+const manufacturers = manufacturersModel(connection, Sequelize)
+const vehicleModels = vehicleModelsModel(connection, Sequelize)
+const myCars = myCarsModel(connection, Sequelize)
+const myCars2Vehicles = myCars2VehiclesModel(connection, Sequelize, myCars, vehicleModels)
```

Comment:
This table isn't necessary, myCars to vehicleModels is actually a 1:1 relationship. Each entry in myCars is talking about a single car.

Reply:
This tale becomes necessary when you add a different vehicle with the same model name.

Reply:
It's still a 1:Many. It would only be a many to many if you were saying 

`"year": "1980", "currentVehicle": "yes", "repurchase": "yes", "models": [3, 5]`
`"year": "1981", "currentVehicle": "yes", "repurchase": "yes", "models": [2, 3, 4]`

But your data would actually be more accurately modeled as

`"year": "1980", "currentVehicle": "yes", "repurchase": "yes", "model": 3`
`"year": "1980", "currentVehicle": "yes", "repurchase": "yes", "model": 5`
`"year": "1981", "currentVehicle": "yes", "repurchase": "yes", "model": 2`
`"year": "1981", "currentVehicle": "yes", "repurchase": "yes", "model": 3`
`"year": "1981", "currentVehicle": "yes", "repurchase": "yes", "model": 4`

The fact that 3 happens twice doesn't make it a many to many. It's only a many to many if a single row can have many.